dle/search/static/search/es_search.js

```diff
@@ -56,7 +56,7 @@ const sk = new Searchkit({
 })
 
 async function vectorizeText(query) {
-    const response = await fetch("http://localhost:8000/api/v1/vectorize", {
+    const response = await fetch("http://API_ENDPOINT:8000/api/v1/vectorize", {
```

Comment:
Can't we use env variable here as well?

Reply:
no, because it is running client side. So, the os.env call does not work.

We can hardcode our LB or domain. Or, set a var.